website/docs/docs/build/python-models.md

```diff
@@ -345,6 +345,7 @@ def model(dbt, session):
     # apply our function
     # (columns need to be in uppercase on Snowpark)
     df["IS_HOLIDAY"] = df["ORDER_DATE"].apply(is_holiday)
+    df["ORDER_DATE"].dt.tz_localize('UTC') #change date type
```

Comment:
Wonderful fix @ernestoongaro!

Based on [these](https://pandas.pydata.org/docs/reference/api/pandas.Series.dt.tz_localize.html) docs, what do you think about this comment?
```suggestion
    df["ORDER_DATE"].dt.tz_localize('UTC') # convert from Number/Long to tz-aware Datetime
```